src/Examples/Flowsheets/ngcc/soec.py

```diff
@@ -1,1233 +1,1265 @@
+            destination=self.makeup_mix.w1,
+        )
+        self.water05 = Arc(
+            doc="Water from heater 2 to mix",
+            source=self.water_heater02.tube_outlet,
+            destination=self.makeup_mix.w2,
+        )
+        self.hstrm04 = Arc(
+            doc="",
+            source=self.feed_hx01.shell_outlet,
+            destination=self.water_heater01.shell_inlet,
+        )
+        self.hstrm06 = Arc(
+            doc="",
+            source=self.h2_drop_water_port,
+            destination=self.h2_precooler.inlet,
+        )
+        self.hstrm07 = Arc(
+            doc="",
+            source=self.h2_precooler.outlet,
+            destination=self.cmp01.inlet,
+        )
+        self.hstrm08 = Arc(
+            doc="",
+            source=self.cmp01.outlet,
+            destination=self.ic01.inlet,
+        )
+        self.hstrm09 = Arc(
+            doc="",
+            source=self.ic01.outlet,
+            destination=self.cmp02.inlet,
+        )
+        self.hstrm10 = Arc(
+            doc="",
+            source=self.cmp02.outlet,
+            destination=self.ic02.inlet,
+        )
+        self.hstrm11 = Arc(
+            doc="",
+            source=self.ic02.outlet,
+            destination=self.cmp03.inlet,
+        )
+        self.hstrm12 = Arc(
+            doc="",
+            source=self.cmp03.outlet,
+            destination=self.ic03.inlet,
+        )
+        self.hstrm13 = Arc(
+            doc="",
+            source=self.ic03.outlet,
+            destination=self.cmp04.inlet,
+        )
+        self.hstrm14 = Arc(
+            doc="",
+            source=self.cmp04.outlet,
+            destination=self.ic04.inlet,
+        )
+        self.hstrm15 = Arc(
+            doc="",
+            source=self.ic04.outlet,
+            destination=self.cmp05.inlet,
+        )
+        self.hstrm16 = Arc(
+            doc="",
+            source=self.cmp05.outlet,
+            destination=self.ic05.inlet,
+        )
+        self.hstrm17 = Arc(
+            doc="",
+            source=self.ic05.outlet,
+            destination=self.cmp06.inlet,
+        )
+        pyo.TransformationFactory("network.expand_arcs").apply_to(self)
+
+    def _add_constraints(self):
+        cost_base.register_idaes_currency_units()
+        self.water_price = pyo.Var(
+            initialize=9.352e-4,
+            units=pyo.units.USD_2018 / pyo.units.kg,
+            doc="Price of water including treatment chemicals",
+        )
+        self.water_price.fix()
+        self.hydrogen_product_rate = pyo.Var(
+            self.config.time,
+            initialize=5,
+            units=pyo.units.kg / pyo.units.s,
+            doc="Hyrogen mass product rate.",
+        )
+
+        @self.Constraint(self.config.time)
+        def hydrogen_product_rate_eqn(b, t):
+            return (
+                b.hydrogen_product_rate[t]
+                == b.cmp06.control_volume.properties_out[0].flow_mass
+            )

```

Comment:
`b.cmp06.control_volume.properties_out[0].flow_mass` to `b.cmp06.control_volume.properties_out[t].flow_mass`